vuepress/docs/next/tutorials/jhipster/jhipster.md

```diff
@@ -0,0 +1,149 @@
+
+## Build and Deploy
+This section will walk you through building a docker image from your microservice, creating your Entando bundle, checking your bundle into git, and deploying it to the Entando Component Repository.
+
+You'll need:
+  - Docker
+  - A running Entando instance (see [Getting started](../../docs/getting-started) for steps if needed)
+  - A bash shell
+  - git
+  - An empty git repository
+
+### Build Docker Image for Microservices
+1. In your microservice project on a command line run `./mvnw -Pprod clean package jib:dockerBuild`
+
+> **Note**
+>
+> By default the organization used to generate the docker image is `entando`, but you can provide a custom value during project initialization, as well as by changing the `pom.xml` file or by providing the `-Djib.to.image=<org>/<name>:<version>` to the `jib:dockerBuild` command.
+
+> **Note**
+>
+> Output image name is generated using the organization value defined during project initialization. You can override the provided values by altering the `pom.xml` file or by customizing the `-Djib.to.image` parameter used in the `./mvnw` command
+
+> **Warning**
+>
+> If you manually override the target image of the docker build, remember to update the plugin metadata in the bundle accordingly in the bundle steps.
+
+2. View your image and tag with `docker images`
+3. Publish the Docker image to Docker registry (DockerHub or equivalent `docker push <name-of-the-image:tag>`
+  > **Note**
+  >
+  > The first time your run this command it will have to push all of the layers. Subsequent runs will be much faster
+
+### Build your Bundle and publish to git
+1. In your microservice project populate the bundle with the generated micro frontends, run the `./buildBundle.sh` script or use `npm run populate-bundle`
+
+> **Important**
+> The bundle population with the micro frontends requires some time to be processed. You should be able to follow the progress of the operation on screen.
+
+2. The output of your bundle will be in the top level `bundle` folder in your microservice
+
+3. Create a new git repository and name it `my-bundle` (or a name of your choice)
+
+4. Clone the repository created in the prior step to a location of your choice (not inside your microservice) `git clone <url of your repo>`
+
+5. Copy the contents of the `bundle/` folder from your microservice into your repository
+
+> **Important**
+> The file descriptor.yaml should be at the top of your repository
+
+6. `cd` into your cloned bundle repository on the command line
+
+6. Add the files to git `git add .`
+
+7. Commit the files to git `git commit -a -m "Your commit message here"`
+
+8. Push the files to git `git push`
+
+9. Tag your bundle `git tag -a "v0.0.1" -m "My first tag"`
+
+10. Push the tags `git push --tags`
+
+11. Install the `entando-bundle-cli` using `npm install -g entando-bundle-cli@6.2.0`
```

Comment:
I know 6.2.0 doesn't exist yet but I had to use this syntax (extra @entando) to install this: npm install -g @entando/entando-bundle-cli@6.1.0. Do we need that here?

Reply:
Yeah. We'll have to confirm this once the release is done.